### ISSUE TITLE
Shipping type/option not shown on invoice

### ISSUE THREAD

**Issue by leesbian** [mageplus/mageplus#20]
Hi, we are reviewing the invoice layout and suddenly we realized that on all invoice the shipping option text is not showing. We see the price for shipping (in our case 5,80) but before the colon there is no text

Imported from : http://www.magentocommerce.com/bug-tracking/issue?issue=13655

**Comment by encryptedband** [comment]
Has anyone been able to reproduce this? I've checked a fairly stock install and the printable, emailed and admin/frontend invoices all look good. So I'm not sure if I'm confused on where this issue is or if their magento was configured wrong?

**Comment by leesbian** [comment]
Thanks!

I've just confirmed via the demo instance that this issue doesn't exist. I've checked both the emails and the PDFs

Likely that as you say, it's a bad configuration or even a bad shipping module

FWIW, all issues have a workflow of initially being "unconfirmed" until somebody has been able to replicate them - then they become "confirmed", which indicates that the issue should be addressed. If like this issue, it has been proven to be invalid, the label "invalid" will be added, and then the issue closed.

I'll write up the issue reporting/resolution procedure in the wiki soon
